src/clapsolver/dynamic_lsap.py

```diff
@@ -0,0 +1,261 @@
+import numpy as np
+
+
+def augment(cost_matrix, curRow, row4col, col4row, u, v):
+    """Find the shortest augmenting path for the selected row. Update the dual
+       variables. Augment the previous solution.
+
+    Parameters
+    ----------
+    cost_matrix : 2darray
+         A matrix of costs.
+    curRow : int
+        Index of the row from which the augmenting starts.
+    row4col: 1darray
+             Specify the rows to which each column is matched with.
+    col4row: 1darray
+             Specify the columns to which each row is matched with.
+    u : 1darray
+        The dual cost vector for rows.
+    v : 1darray
+        The dual cost vector for columns.
```

Comment:
Sometimes dual variables are `u` and `v`, sometimes they are `alpha` and `beta` in the literature. I think `u` and `v` are fine since you explain in the docstring which one is which. I'd add a comment in the docstring for each along the lines of
```
    u : 1darray
        The dual cost vector for rows. AKA ``α`` in some papers.
    v : 1darray
        The dual cost vector for columns. AKA ``β`` in some papers.
```